<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络调试助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="ServertextEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1,0,1">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string>通信协议</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxTcp">
          <item>
           <property name="text">
            <string>TCP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UDP</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>服务器IP地址</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxIP"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>端口号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="NetworkComboBox" name="comboBoxChildren"/>
        </item>
        <item>
         <widget class="QPushButton" name="Startconnect">
          <property name="text">
           <string>开始监听</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Stopconnect">
          <property name="text">
           <string>停止监听</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="breakconnect">
          <property name="text">
           <string>断开</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,1">
        <item>
         <widget class="QTextEdit" name="ClienttextEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="SendButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NetworkComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">../../../networkcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
